css/style.css

```diff
@@ -432,18 +440,33 @@ th {
 }
 
 .container {
-  width: 60%;
+  width: 65%;
   display: flex;
   justify-content: space-between;
 }
 
+@media (max-width: 56.25em) {
+  .container {
+    width: 100%;
+    padding: 0 1rem;
+    justify-content: space-around;
+  }
+}
+
 .navigation {
-  width: 65%;
+  width: 75%;
   display: flex;
   justify-content: center;
   align-items: center;
 }
 
+@media (max-width: 37.5em) {
+  .navigation {
+    opacity: 0;
+    visibility: hidden;
+  }
```

Comment:
I suppose you did this in anticipation of collapsing your menu? 🙂

Reply:
Yes, that is right. I will fix it later.